src/squeezed.ml

```diff
@@ -48,6 +48,7 @@ let _ =
 
 	let rpc_server = Thread.create Xcp_service.serve_forever server in
 
+	Memory_server.record_boot_time_host_free_memory ();
```

Comment:
Put this before rpc_server thread is created (line 49) because squeezed should record the memory before making it possible for clients to request the reading of that value. If the recording happens after then it makes it possible for a race condition to occur.